plugins/by-name/rest/default.nix

```diff
@@ -230,63 +233,63 @@ helpers.neovim-plugin.mkNeovimPlugin {
 
         default: `{}`
       '';
-      apply = v: if isAttrs v then mapAttrs (_: helpers.mkRaw) v else v;
+      apply = v: if lib.isAttrs v then lib.mapAttrs (_: lib.nixvim.mkRaw) v else v;
```

Comment:
We really must get around to moving this to the type itself. Having to do this in every option that uses strLua is too awkward.

`coercedTo str mkRaw rawLua // { description = ""; }` is all we need :sob:

Reply:
Just changing the definition from 
```nix
  strLikeType =
    description:
    lib.mkOptionType {
      name = "str";
      inherit description;
      descriptionClass = "noun";
      check = v: lib.isString v || isRawType v;
      merge = lib.options.mergeEqualOption;
    };
```

Reply:
Not in this PR of course, but this will be the first step before #2003